src/app/hooks/router/useExploreSelected.ts

```diff
@@ -11,7 +11,7 @@ export const useExploreSelected = (): boolean => {
   return !!match;
 };
 
-export const useExploreFeaturedSelected = (): boolean => {
+export const useExploringFeaturedRooms = (): boolean => {
```

Comment:
is there any specific reason for changing it's name? previous naming strategy is used because it clearly describe the section Explore > Featured (selected usage).